PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,22 @@
-# Pull Request Guidelines
-
-Pull requests are an essential collaborative tool for modern software development. 
-
-The below list is intended to help you figure out whether your code is ready to be reviewed
-and merged into ArangoDB. The overarching goal is to:
-
-- Reduce the amount of *recurring* defects
-- Reduce the impact to the other developer’s time and energy spent on defects in your code
-- Increase the overall autonomy and productivity of individual developers
-
-## Acceptance Checklist
-
-*The below list is not exhaustive, think thoroughly whether the provided information is sufficient.*
-*Remove options that do not apply*
-
 ### Scope & Purpose
 
-*(Can you describe what functional change your PR is trying to effect?)*
+*(Please describe the changes of this PR for reviewers)*
 
-- [ ] Bug-Fix for *devel-branch* (i.e. no need for backports?)
-- [ ] Bug-Fix for a *released version* (did you remember to port this to all relevant release branches?)
```

Comment:
I did like these, for one to check whether backports are done, and to link all backport PRs here.

I don't need these two checkboxes as they were, but can we have something for backports?

Reply:
Sure, let's add backports somewhere!